### DISCUSSION TITLE
prisma and nestjs for autmate from define entety to rest api

### DISCUSSION THREAD

**Discussion by gitsupi** [prisma/prisma#10785] [Q&A]

**Comment by ghost** [comment]
This might need to be done by 3rd party generators. But so far I am unable to find one for REST API that does the job.

**Comment by ghost** [comment]
You can search `prisma nextjs dto generator` and look if they would fit your needs. But personally it's a bit limited.

**Comment by Jaikant** [comment]
This starter kit could also be useful [https://github.com/notiz-dev/nestjs-prisma-starter](https://github.com/notiz-dev/nestjs-prisma-starter)
